screwdriver.yaml

```diff
@@ -0,0 +1,47 @@
+shared:
+    image: node:8
+jobs:
+    SIMPLE:
+        steps:
+            - echo: echo test
+        requires: [~pr, ~commit]
+    PARALLEL1:
+        steps:
+            - echo: echo test
+        requires: [SIMPLE]
+    PARALLEL2:
+        steps:
+            - echo: echo test
+        requires: [SIMPLE]
+    JOIN:
+        steps:
+            - echo: echo test
+        requires: [PARALLEL1, PARALLEL2]
+    FAIL:
+        steps:
+            - echo: exit 1;
+        requires: [~pr, ~commit]
```

Comment:
This should be only `~commit`
https://github.com/screwdriver-cd/screwdriver/blob/9e917c4419485e4a844f29533998dc9f6605b337/features/workflow.feature#L22

Reply:
Thanks, fixed it.